views/project/user.erb

```diff
@@ -77,54 +77,48 @@
           </thead>
           <tbody class="divide-y divide-gray-200 bg-white">
             <% @users.each do |user| %>
-              <tr id="user-<%= user[:ubid]%>">
+              <tr id="user-<%= user.ubid %>">
                 <td class="whitespace-nowrap py-4 pl-4 pr-3 text-sm font-medium text-gray-900 sm:pl-6" scope="row">
-                  <%= user[:email] %>
+                  <%= user.email %>
```

Comment:
We typically use serializers as a layer between `views` and `routes` to manage the data we send to the web or API. Directly passing models or logic to the view can complicate data management. Although it might violated a few times, it's the current pattern in UI/route relationships. 

What's your opinion? Should we continue with this method or ditch it?

Reply:
We should definitely ditch the use of serializers for the web routes, where they are an unnecessary and IMO negative abstraction. I think their use in the web routes makes the code worse in all cases where they are used.

I would recommend only using serializers in api routes, and only if multiple routes use the same serialization, where the abstraction makes sense.  Otherwise, the serialization code can be inlined into the single route that uses it.  Further, I recommend that even in that case, the serializer code be moved to Clover helper methods.

However, I'm not advocating we make this change all at once.  We can certainly do it piecemeal.